### ISSUE TITLE
[Feature request] Wired host mode

### ISSUE THREAD

**Issue by asimonf** [ruundii/bthidhub#32]
I know Raspikey exists and this is supposed to be a wireless alternative. However, Raspikey only works with keyboards. This tool is much more useful in my scenario, but I need wired support rather than wireless. 

Thanks for your work. Cheers!

**Comment by Dreamsorcerer** [comment]
I suspect nobody is going to volunteer to work on such a feature. You'd probably find it easier to hack raspikey to use a different HID descriptor that works with a mouse.

**Comment by asimonf** [comment]
I suspect that's true. I'm already giving it a try. Though I don't own a RPi Zero, I noticed that the code doesn't really have anything too specific to it and should work on debian with few modifications. I'm trying it on a Beaglebone right now with a USB BT adapter and since that's what I have in mind.

**Comment by Dreamsorcerer** [comment]
Indeed, it's only really the automated install that likely requires a RPi Zero to work out of the box.

**Comment by asimonf** [comment]
Thanks for the impulse. I ended up making a prototype that works pretty well on a Beaglebone and even added support for Gamepads, though that requires an app on the PC side of things.

**Comment by technikjunge** [comment]
I would also find this feature very useful.  How did you implement this exactly @asimonf ? Did you publish the code?
